ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -199,7 +197,7 @@ class SimpleOperationTracker implements OperationTracker {
     totalOfflineReplicaCount =
         getReplicasByState(null, EnumSet.of(ReplicaState.OFFLINE)).getOrDefault(ReplicaState.OFFLINE,
             Collections.emptyList()).size();
-    allReplicas = allDcReplicasByState.values().stream().flatMap(Collection::stream).collect(Collectors.toList());
+    allReplicas = partitionId.getReplicaIds().stream().collect(Collectors.toList());
```

Comment:
The test case didn't cover this change. Can you add some test regarding this change(for example, one replica not found, one replica no state, it will do cross colo ttl update? So later when people accidentally change it back or change to something else, it will trigger the tests.

Reply:
Done in latest commit.